source/whats-new.txt

```diff
@@ -32,6 +33,21 @@ Learn what's new in:
 * :ref:`Version 1.1 <version-1.1>`
 * :ref:`Version 1.0 <version-1.0>`
 
+.. _version-1.12.1:
+
+What's New in 1.12.1
```

Comment:
In other versions, patches are presented as Important admonitions inside the minor version. I didn't do that here because 1.12 already starts with such an admonition.